screens/rewards/RewardsScreen.js

```diff
@@ -170,7 +175,13 @@ export default class RewardsScreen extends React.Component {
               onRefresh={this._onRefresh}
             />
           }>
-          <Text styles={{ fontSize: 24 }}> Healthy Rewards </Text>
+          <TouchableOpacity
```

Comment:
Works fine but not an intuitive trigger from a user stand-point since there is no icon + the trigger area is actually quite large! According to our current design/Figma, it seems like there should be an arrow down button that triggers it - possible stream-lining with the News cross button that you created?

Reply:
Also stretch: making the screen transition between screens smoother somehow